source/todo.rst

```diff
@@ -1,6 +0,0 @@
-.. _todo-label:
```

Comment:
This file is important and should not be removed.

Reply:
restored.